src/Forms/Form/Form.stories.mdx

```diff
@@ -57,10 +87,538 @@ const fields = useMemo(
+	[]
+);
+```
+
+
+## FormFieldDateField
+- It allows the user to view and pick dates from a calendar widget or manually type the date in the text field.
+- [**Playground**](/?path=/story/formfields-formfielddatefield--playground)
+- Default Value: `string` - Date in UTC format transformed to string.
+
+### Props (Input Settings)
+* **showTime** - `boolean` optional - When true a time field will appear next to the date field.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'date',
+				inputSettings: {
+					showTime: false,
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+## FormFieldRadioButton
+- A group of radio buttons allows users to select a single item from a list of possible options.
+- All possible options are exposed up front for comparison.
+- Users can only make a single selection from the list of possible options.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldradio--playground
+
+#### Props (Input Settings)
+* **options** - `array` of `object` optional - Predefined set of mutually exclusive options.
+	* **label** - `string` required - Description of the option.
+	* **value** - `string` required - Value of the option, this is stored in the Form state.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+		() =>
+			[
+				//...other fields
+				{
+					//...all generic field props,
+					type: "radio",
+					inputSettings: {
+						options: [
+							{
+								label: "Label 1",
+								value: "label_1",
+							},
+							{
+								label: "Label 2",
+								value: "label_2",
+							},
+							{
+								label: "Label 3",
+								value: "label_3",
+							},
+						]
+					},
+					helperText: 'Helper text',
+					instructionText: 'Instruction text'
+				}
+				//...other fields
+			] as FieldDef<FormFieldRadioDef>[],
+		[]
+	);
+```
+
+## FormFieldText
+- Text input fields are used to enter text information, numbers, e-mail addresses, or passwords. 
+- A text field consists of a label and a line with variations in width.
+- [**Playground**](/?path=/story/formfields-formfieldtext--playground)
+
+#### Props (Input Settings)
+
+* **htmlFor** - `string` optional - Specifies which form element a label is bound to.
+* **maxCharacters** - `number` optional - Defines the limit of characters that are allow to type in the input element.
+* **multiline** - `boolean` optional - When is enabled the text field will expand its height.
+* **placeholder** - `string` optional - The hint displayed in the input before the user enters a value.
+* **prefixElement** - `JSX.Element` optional - Icon at the beginning of the text field.
+* **size** - `Size | string` required - Size of the field. Could be either one of the predefined sizes (please see Sizes section), or a custom size. This property can be internally overwritten if the field is placed inside of a row with other fields.

```

Comment:
Shouldn't size be optional? We already have a functionality that if no size gets passed then a default takes place, please check this.

Reply:
Also I'm thinking we could remove size as an inputSettings prop and just inject it directly in Col.tsx (inject it from fieldDef), but this is something for another ticket.